src/boat/private/captain/manifest.nim.ini

```diff
@@ -0,0 +1,23 @@
+; FYI
+; 0 = false, 1 = true
+
+; captains section
+; each line represents an A, B or C to load
+;   A. dir path containing a `manifest.nim.ini`
+;   B. file path pointing to `manifest.nim.ini`
+;   C. a URL pointing to a `manifest.nim.ini`
+; TODO: cmd to parse arbitrarily, e.g. boat conf load ./some/dir
+; TODO: cmd to see parsed captains: e.g. boat conf list -> X, Y, Z
+[captains]
+; TODO: confirm $HOME can be parsed to absolute path
+; ^ FYI: $HOME wont be available for accounts without a $HOME dir (e.g. some sys accounts)
```

Comment:
check implementation of nimlangs getHomeDir() which may have already solved this problem